pom.xml

```diff
@@ -70,6 +71,8 @@
     <hsqldb.version>2.7.1</hsqldb.version>
     <!-- should match dropwizard version - except for security issues -->
     <jackson.version>2.18.1</jackson.version>
+    <!-- Skip Jacoco by default -->
+    <jacoco.skip>true</jacoco.skip>
```

Comment:
what's the test time difference between jacoco enabled and disabled ? 
if it's not too much maybe we could have it run in all PRs

Reply:
just 3 second difference
lemme remove the flag